src/main/java/uk/gov/gchq/palisade/resource/impl/DirectoryResource.java

```diff
@@ -57,6 +57,26 @@ public void setParent(final ParentResource parent) {
         parent(parent);
     }
 
+    @Override
+    public boolean equals(final Object o) {
```

Comment:
Has the change in way of doing equals and hashcode methods been agreed by the team?